lxd/instance/drivers/driver_qemu.go

```diff
@@ -5157,3 +5157,40 @@ func (vm *qemu) writeInstanceData() error {
 
 	return nil
 }
+
+// Info returns "qemu" and the currently loaded qemu version, 0 if not found.
+func (vm *qemu) Info() instance.Info {
+	var qemuVersion, qemuBinary, qemuPath string
+	var out []byte
+	var err error
+	data := instance.Info{
+		Name:    "qemu",
+		Version: "0",
+	}
+
+	if vm.architecture == 0 {
+		vm.architecture, err = osarch.ArchitectureGetLocalID()
+	}
+	if err != nil {
+		return data
+	}
+
+	qemuBinary, _, err = vm.qemuArchConfig()
+	if err != nil {
+		return data
+	}
+
+	qemuPath, err = exec.LookPath(qemuBinary)
+	if err != nil {
+		return data
+	}
```

Comment:
Instead of duplicating this part in both `Start()` and `Info()`, it may be better to just have `qemuArchConfig()` call `exec.LookPath` itself.

Reply:
That part and my earlier comment about passing the architecture directly to `qemuArchConfig` still need addressing.
We shouldn't have any of this `d.architecture` or `qemuPath` business in here, we should instead just have:

```
hostArch, err := osarch.ArchitectureGetLocalID()
if err != nil {
    return data
}

qemuPath, err := d.qemuArchConfig(hostArch)
if err != nil {
    return data
}
```

Reply:
Implemented in most recent commit